active-rfcs/0009-global-api-change.md

```diff
@@ -44,6 +46,8 @@ app.mixin(/* ... */)
 app.component(/* ... */)
 app.directive(/* ... */)
 
+app.config.globalProperties.customProperty = () => {}
```

Comment:
Is there an interface that can be extended to type new properties?

Reply:
I'd say just don't use this if using TS.

Reply:
Wouldn't this be better for people migrating TS projects using libraries that are not adapted yet (or that may never completely adapt) to the inject/provide?

Reply:
There's no way to make this work in TS though. If users insist on using this with TS they'd have to shim the types themselves.

Reply:
> If users insist on using this with TS they'd have to shim the types themselves

Oh yeah, I just thinking of exposing an interface that can be extended by users to add their own typings